### DISCUSSION TITLE
[2817530]恋恋女团 {游戏本体}

### DISCUSSION THREAD

**Discussion by ali11515** [pjy612/SteamManifestCache#674] [Share]
### Steam 游戏ID

2817530

### 密钥列表

```hash
2817531;d16926e4133c6e84a277b2d7c050cf83af81d87eeaa1ffd39ff646cbb7cdb20f
```

### 清单附件

[[2817530]恋恋女团 {游戏本体}.zip](https://github.com/pjy612/SteamManifestCache/files/15402084/2817530.zip)

### 备注

code:7D3AA2B2355C96AA
email:[20583283@qq.com](mailto:20583283@qq.com)

**Comment by ali11515** [comment]
请求加入内侧~~

**Reply by pjy612** [reply]
没有内测

**Reply by finding48** [reply]
大佬 ，项目停了吗
